schematools/introspect/db.py

```diff
@@ -9,6 +9,7 @@
 
 DB_TO_ASCHEMA_TYPE = {
     "DATE": {"type": "string", "format": "date"},
+    "TIME": {"type": "string", "format": "time"},
```

Comment:
Vergeet dan niet mapping andersom maken, zoals hier:
https://github.com/Amsterdam/schema-tools/pull/13/files#diff-bef99fb4d8d02de3ede325589290b918R51-R52

Reply:
Da's een goeie! Ik wacht nog even met gebruiken tot PR 13 er door is!